src/libclipper/include/clipper/redis.hpp

```diff
@@ -24,7 +24,7 @@ const std::string LOGGING_TAG_REDIS = "REDIS";
 /**
  * Issues a command to Redis and checks return code.
  * \return Returns true if the command was successful.
-*/
+ */
```

Comment:
I think there's something wrong with the way this was formatting things. This shouldn't change.